src/app-layer-protos.h

```diff
@@ -57,6 +57,7 @@ enum AppProtoEnum {
     ALPROTO_TEMPLATE_RUST,
```

Comment:
Is there no new detection ?

Reply:
I may have used the wrong terminology in my description in the PR. My intentions for the PR are more so to display the app_proto for torrent traffic as "bittorrent_dht" instead of showing up as "failed", and also parse and log the bittorrent_dht app-layer protocol to eve.json. If someone feels that detection keywords for bittorrent-dht would be useful could we create another issue or PR to add these?

Reply:
Okay, that can be an additional work for later